### ISSUE TITLE
dynamic as2_names for several same type sensors

### ISSUE THREAD

**Issue by pariaspe** [aerostack2-developers/as2_core#28]
Two same type sensor will publish now at the same topic (`/drone_sim_0/sensor_measurements/gps`) following the current name convention.
```json
"model": "quadrotor_base",
"name": "drone_sim_0",
"payload": {
    "gps0": {
        "sensor": "gps"
    },
    "gps1": {
        "sensor": "gps"
    }
}
```
Shouldn't they publish on different topics `/drone_sim_0/sensor_measurements/{sensor_name}/gps`?

**Comment by miferco97** [comment]
yes, but we have to think about how to do it, e.g., I would not use `/ns/sensor_measurements/gps0/gps` but for cameras `/ns/sensor_measurements/camera1/image_raw` sounds good to me. In imu/gps/... measurements maybe i would put `/ns/sensor_measurements/gps1` or `/ns/sensor_measurements/gps1` directly. What do you think @perezsaura-david @pariaspe  @RPS98? The bad thing about this is that we have to make distinctions between these sensors so maybe using  topics like `/drone_sim_0/sensor_measurements/gps0/gps` is more general.
